### ISSUE TITLE
MooseX-App-1.41 t/05_extended.t is failing

### ISSUE THREAD

**Issue by XSven** [maros/MooseX-App#62]
perl 5.14.2
platform AIX 7.2

```
t/05_extended.t ........... 1/30
    #   Failed test 'Check enum error message'
    #   at t/05_extended.t line 244.
    #          got: 'Validation failed for '__ANON__' with value ggg. Value must be equal to "aaa", "bbb", "ccc", "ddd" "eee", or "fff".'
    #     expected: 'Value must be one of these values: aaa, bbb, ccc, ddd, eee, fff (not 'ggg')'
    # Looks like you failed 1 test of 2.

#   Failed test 'Test enum error message'
#   at t/05_extended.t line 245.
```

**Comment by jplesnik** [comment]
The failure is related to the [change](https://github.com/moose/Moose/commit/c554e018b47487e774e71d6022033bffb5293ba2) in Moose 2.2014

**Comment by XSven** [comment]
Stringified exceptions should never be assumed to be part of the (stable) interface of a given framework like for example Moose. The above unit test should either be removed or redesigned.

**Comment by eserte** [comment]
The test as such should stay, it should just be changed to accept both error message variants, old and new.

**Comment by maros** [comment]
Released as 1.42
